cultcargo/casa/setjy.yml

```diff
@@ -13,12 +13,36 @@ cabs:
                 - lib.params.casa.ms-selectdata-inputs
             _scrub: antenna, uvrange, msselect   # not used by this task
             standard:
-                dtype: str 
-                info: |
-                    Which standard to use. Default is 'Perley-Butler 2017', alternatives are 
-                    'Perley-Butler 2013', 'Perley-Butler 2010', 'Butler-JPL-Horizons 2012', 
-                    'manual', 'fluxscale'
+                choices:
+                    - 'Perley-Butler 2017'
+                    - 'Perley-Butler 2013'
+                    - 'Perley-Butler 2010'
+                    - 'Skaife-Heald 2012'
+                    - 'Butler-JPL-Horizons 2010'
+                    - 'Butler-JPL-Horizons 2012'
+                    - 'Stevens-Reynolds 2016'
+                    - manual
+                    - fluxscale
+                dtype: str
+                default: 'Perley-Butler 2017'
+                info: Which standard to use 
+            scalebychan:
+                dtype: bool
+                default: true
```

Comment:
Same as above, I would omit the default.

Reply:
Done